drivers/storage/portworx/components_test.go

```diff
@@ -3156,6 +3163,8 @@ func TestAutopilotInstall(t *testing.T) {
 		},
 	}
 
+	cluster := stcSpec
```

Comment:
nit: This is not copying the object but just assigning the pointer to another variable. It will result in modification of both objects. If you really want to copy the object use `DeepCopy` here and on line 3238.
```
cluster := stcSpec.DeepCopy()
```

Reply:
You're right. We don't want the original object messed up by copying it. DeepCopy is required here. Fixed it. Thanks